network/p2p/fixture.go

```diff
@@ -0,0 +1,250 @@
+package p2p
```

Comment:
Spoiler alert: this file does not have new features, just moved and refactored all shared fixtures among our tests for better organization, readability, and maintainability.